src/pages/lti/deepLinkContent.js

```diff
@@ -0,0 +1,264 @@
+                    <InputGroup className="mb-3">
+                        <InputGroup.Prepend className="content_type_name">
+                            <InputGroup.Text>{data.name} </InputGroup.Text>
+                        </InputGroup.Prepend>
+                        <FormControl value={count} onChange={e => {
+                            const newValue = e.target.value
+                            setCount(newValue)
+                            setValue(data.key, newValue, items)
+
+                        }} onInput={event => {
+                            const keyCode = event.keyCode || event.which;
+                            const keyValue = String.fromCharCode(keyCode);
+                            if (/\+|-/.test(keyValue))
+                                event.preventDefault();
+                        }}
+                        />
+                    </InputGroup>
+                </div>
+                <Button variant="danger" onClick={() => {
+                    if (count > 0) {
+                        setCount(count - 1)
+                        decrement(data.key, items)
+                    }
+                }}>
+                    <ShieldMinus/>
+                </Button>
+
+
+                <Button variant="success" onClick={() => {
+                    setCount(count + 1)
+                    increment(data.key, items)
+                }}>
+                    <Plus/>
+                </Button>
+            </Card.Body>
+        </Card>
+    );
+
+}
+
+/**
+ * Request handler
+ * @param items
+ * @returns {JSX.Element}
+ * @constructor
+ */
+function LtiContentRequest({items}) {
+    const [error, setError] = useState(null);
+    const [isLoaded, setIsLoaded] = useState(false);
+    const [response, setResponse] = useState(null)
+
+    /**
+     *
+     * @returns {Promise<void>}
+     */
+    async function getLTIContent() {
+
+        toolClient.getLTIContent(items).then()
+            .then(
+                (result) => {
+                    setIsLoaded(true);
+                    setResponse(result)
+                },
+                // Note: it's important to handle errors here
+                // instead of a catch() block so that we don't swallow
+                // exceptions from actual bugs in components.
+                (error) => {
+                    setIsLoaded(true);
+                    setError(error);
+                }
+            )
+    }
+
+    async function doFormPost() {
+
+    }
+
+    if (error) {
+        return <div>Error: {error.message}</div>;
+    } else if (!isLoaded) {
+        return (
+            <Button onClick={getLTIContent}>
+                Request
+            </Button>
+        );
+    } else {
+        return (
+            <Card>
+                <Card.Body>
+                    <JsonAccordion json={items} title={'Items'}/>
+                    <Button onClick={getLTIContent}>
+                        Update
```

Comment:
We should think about localizing this app

Reply:
I just added a localization functionality